odoo/modules/loading.py

```diff
@@ -164,7 +165,11 @@ def load_test(idref, mode):
         if skip_modules and module_name in skip_modules:
             continue
 
-        _logger.debug('loading module %s (%d/%d)', module_name, index, module_count)
+        init_module_query_count = cr.sql_log_count
+        module_t0 = time.time()
+        module_t0_sql = odoo.sql_db.sql_counter
+
+        _logger.info('Loading module %s (%d/%d)', module_name, index, module_count)
```

Comment:
You raise this logging level from `debug` to `info`.
Isn't there a risk of logfile blowup?